pkg/controller/job/job_controller_test.go

```diff
@@ -1502,6 +1580,8 @@ func TestTrackJobStatusAndRemoveFinalizers(t *testing.T) {
 	}
 	for name, tc := range cases {
 		t.Run(name, func(t *testing.T) {
+			defer featuregatetesting.SetFeatureGateDuringTest(t, feature.DefaultFeatureGate, features.ScalableIndexedJob, true)()
```

Comment:
remove this

Reply:
woops, i meant to only include the feature gate in the PR with the validation change. it is now removed.